static/app/views/settings/projectDataForwarding/index.tsx

```diff
@@ -17,185 +18,188 @@ import {t, tct} from 'sentry/locale';
 import type {TimeseriesValue} from 'sentry/types/core';
 import type {Series} from 'sentry/types/echarts';
 import type {Plugin} from 'sentry/types/integrations';
-import type {RouteComponentProps} from 'sentry/types/legacyReactRouter';
-import type {Organization} from 'sentry/types/organization';
 import type {Project} from 'sentry/types/project';
-import withOrganization from 'sentry/utils/withOrganization';
+import {useApiQuery} from 'sentry/utils/queryClient';
+import useOrganization from 'sentry/utils/useOrganization';
+import {useParams} from 'sentry/utils/useParams';
 import SettingsPageHeader from 'sentry/views/settings/components/settingsPageHeader';
 import TextBlock from 'sentry/views/settings/components/text/textBlock';
 import PermissionAlert from 'sentry/views/settings/project/permissionAlert';
 
-type StatProps = {
-  params: {
-    orgId: string;
-    projectId: string;
+function DataForwardingStats() {
+  const {orgId, projectId} = useParams();
```

Comment:
wish it wasn't optional
```suggestion
  const {orgId, projectId} = useParams<{orgId: string; projectId: string}>();
```